docs/standard/commandline/snippets/customize-help/csharp/Program.cs

```diff
@@ -0,0 +1,183 @@
+﻿// <all>
+using System.CommandLine;
+using System.CommandLine.Builder;
+using System.CommandLine.Help;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+using Spectre.Console;
+
+class Program
+{
+    static async Task Main(string[] args)
+    {
+        await Original(args);
+        await First2Columns(args);
+        await DescriptionSection(args);
+    }
+
+    static async Task Original(string[] args)
+    {
+        // <original>
+        var fileOption = new Option<FileInfo>(
+            "--file",
+            description: "The file to print out.",
+            getDefaultValue: () => new FileInfo("scl.runtimeconfig.json"));
+        var lightModeOption = new Option<bool> (
+            "--light-mode",
+            description: "Determines whether the background color will be black or white");
+        var foregroundColorOption = new Option<ConsoleColor>(
+            "--color",
+            description: "Specifies the foreground color of console output",
+            getDefaultValue: () => ConsoleColor.White);
+
+        var rootCommand = new RootCommand("Read a file")
+        {
+            fileOption,
+            lightModeOption,
+            foregroundColorOption
+        };
+
+        rootCommand.SetHandler(
+            (FileInfo file, bool lightMode, ConsoleColor color) =>
+            {
+                Console.BackgroundColor = lightMode ? ConsoleColor.White: ConsoleColor.Black;
+                Console.ForegroundColor = color;
+                Console.WriteLine($"--file = {file?.FullName}");
+                Console.WriteLine($"File contents:\n{file?.OpenText().ReadToEnd()}");
+            },
+            fileOption,
+            lightModeOption,
+            foregroundColorOption);
+
+        await rootCommand.InvokeAsync(args);
+        // </original>
+        Console.WriteLine("Default help");
+        await rootCommand.InvokeAsync("--light-mode false --color Yellow");
```

Comment:
It may be easier for the user if we include -h here, even if the specific CLI design does not require it.

Reply:
It's a Friday afternoon, but doesn't this print in the correct order due to oddities in async?